### ISSUE TITLE
FE - Monitor Config Pages

### ISSUE THREAD

**Issue by ajhollid** [bluewave-labs/checkmate#1304]
Monitor Config pages and Create Monitor pages are nearly identical.  It would probably be best to refactor to reuse the "Create" page as "Configure" page as well.

This applies to Uptime, PageSpeed, and Infrastructure monitors

**Comment by Jesulayomy** [comment]
Hello, I've worked on a solution for merging the `PageSpeedConfig` and `PageSpeedCreate` monitor pages on the PageSpeed Components into a single `PageSpeedSetup` Component. Can I get this issue assigned to me so I can make a PR for you to review?

You can check out what I've worked on so far on my [commit](https://github.com/Jesulayomy/Checkmate/commit/f00d7275e5b977b73762c9a7b8c1e0dc53036a86) and let me know here or on [discord](https://discord.com/users/488238079753060353)

**Comment by gorkem-bwl** [comment]
Thank you @Jesulayomy !

**Comment by Jesulayomy** [comment]
Created PR #2588 to implement PageSpeed monitor refactor

**Comment by Jesulayomy** [comment]
@ajhollid @gorkem-bwl Hello, got this notification and thanks for the help. Do you want to create two other issues for the  Uptime and Infrastructure monitors since my PR #2588 only addressed the PageSpeed refactor? You can mention this Issue or the PR as a related one since it would be another two large PRs.

**Comment by gorkem-bwl** [comment]
> @ajhollid @gorkem-bwl Hello, got this notification and thanks for the help. Do you want to create two other issues for the  Uptime and Infrastructure monitors since my PR #2588 only addressed the PageSpeed refactor? You can mention this Issue or the PR as a related one since it would be another two large PRs.

Yes please! For clarity feel free to create two small issues and ref this one so you can create PRs against. I'll then assign you.

Many thanks!
